website/docs/reference/global-configs.md

```diff
@@ -107,7 +107,34 @@ The exception: Some global configurations are actually set in `dbt_project.yml`,
 
 </VersionBlock>
 
-<VersionBlock firstVersion="1.1">
+<VersionBlock firstVersion="1.5">
+
+### Cache population
+
+At the start of runs, dbt caches metadata about all the objects in all the schemas where it might materialize resources (such as models). By default, dbt populates the cache with information on all schemas related to the project.
+
+There are two ways to optionally modify this behavior:
+- `POPULATE_CACHE` (default: `True`): Whether to populate the cache at all. To skip cache population entirely, use the `--no-populate-cache` flag or `DBT_POPULATE_CACHE: False`. Note that this does not _disable_ the cache; missed cache lookups will run queries, and update the cache afterward.
+- `CACHE_SELECTED_ONLY` (default `False`): Whether to limit cache population to just the resources selected in the current run. This can offer significant speed improvements when running a small subset of a large project, while still providing the benefit of caching upfront.
+
+For example, to quickly compile a model that requires no database metadata or introspective queries:
+```text
+
+$ dbt --skip-populate-cache compile --select my_model_name
+
+```
+
+Or, to improve speed and performance while focused on developing Salesforce models, which are materialized into their own dedicated schema, you could select those models and pass the `cache-selected-only` flag:
+
+```text
+
+$ dbt --cache-selected-only run --select salesforce
+
+```
```

Comment:
I don't know what our style guide says about terminal commands beginning with `$` or not.

But I prefer _without_ because it makes it easier for me to copy-n-paste the entire block without having to carefully exclude the leading character.

In fact, I'd vote for doing reviewing all our code examples throughout docs.getdbt.com and removing all the `$`! 🤑

Reply:
i think our style guide examples dont have the `$` symbol so i agree with @dbeatty10 ! will open an issue!

Reply:
I'm just going to change them all! try and stop me!